Code/Editor/Plugins/EditorAssetImporter/AssetBrowserContextProvider.h

```diff
@@ -25,17 +25,24 @@ namespace AZ
 {
     class AssetBrowserContextProvider
         : public AzToolsFramework::AssetBrowser::AssetBrowserInteractionNotificationBus::Handler
+        , public AzToolsFramework::AssetBrowser::AssetBrowserPreviewRequestBus::Handler
     {
     public:
         AssetBrowserContextProvider();
         ~AssetBrowserContextProvider() override;
      
         /////////////////////////////////////////////////////////////////////////////////////////////////////
         // AzToolsFramework::AssetBrowser::AssetBrowserInteractionNotificationBus::Handler
-        void AddSourceFileOpeners(const char* fullSourceFileName, const AZ::Uuid& sourceUUID, AzToolsFramework::AssetBrowser::SourceFileOpenerList& openers) override;
         AzToolsFramework::AssetBrowser::SourceFileDetails GetSourceFileDetails(const char* fullSourceFileName) override;
         /////////////////////////////////////////////////////////////////////////////////////////////////////
 
+        //////////////////////////////////////////////////////////////////////////
+        // AzToolsFramework::AssetBrowser::AssetBrowserPreviewRequestBus::Handler
+        bool PreviewSceneSettings(const AzToolsFramework::AssetBrowser::AssetBrowserEntry* selectedEntry) override;
+        QMainWindow* GetSceneSettings() override;
+        bool SaveBeforeClosing() override;
+        //////////////////////////////////////////////////////////////////////////
```

Comment:
Nit: Since this is a small file, can you switch to the short-hand instead for here and above?
```suggestion
        // AssetBrowserPreviewRequestBus overrides ...
        bool PreviewSceneSettings(const AzToolsFramework::AssetBrowser::AssetBrowserEntry* selectedEntry) override;
        QMainWindow* GetSceneSettings() override;
        bool SaveBeforeClosing() override;
```

Reply:
Yup, done!